feature_extractor.py

```diff
@@ -123,7 +123,7 @@ def write(self, feature, video_name, idx, dir):
         self.store(feature, idx)
 
 
-def read_features(file_path, cache=None):
+def read_features(file_path, feature_dim, cache=None):
```

Comment:
I added a parameter feature_dim. 
Which is better, passing feature dimension or feature extractor type?

Reply:
This is good